volkszaehler/__init__.py

```diff
@@ -45,3 +50,4 @@ async def get_data(self):
         self.min = self.data["data"]["min"][1]
         self.consumption = self.data["data"]["consumption"]
         self.tuples = self.data["data"]["tuples"]
+        self.last = self.datanow["data"]["average"]
```

Comment:
The latest value is located in the tuples, e.g., `[1598106085887,0.01,1]` and could be extracted from there. As it looks like that the array with the data is probably sorted. Otherwise it would be something like the "last average" which is computed with a single value and this doesn't make much sense.

Reply:
but in my envirement it delivers different data:

url: http://192.168.xx.xx/middleware.php/data/xxxxxx-03df-11e6-xxxx-d181e0721599.json

  |  
-- | --
version | "0.3"
data |  
tuples |  
0 |  
0 | 1603962936115
1 | 1516.93
2 | 1
uuid | "5b5f5e60-03df-11e6-990a-d181e0721599"
from | 1603962876310
to | 1603962936115
min |  
0 | 1603962936115
1 | 1516.9300227527604
max |  
0 | 1603962936115
1 | 1516.9300227527604
average | 1516.93
consumption | 25.2
rows | 2


url:  http://192.168.xx.xx/middleware.php/data/xxxxxx-03df-11e6-xxxx-d181e0721599.json?from=now

  |  
-- | --
version | "0.3"
data |  
tuples |  
0 |  
0 | 1604049337213
1 | 821.001
2 | 1
uuid | "5b5f5e60-03df-11e6-990a-d181e0721599"
from | 1604049276263
to | 1604049337213
min |  
0 | 1604049337213
1 | 821.0008206965983
max |  
0 | 1604049337213
1 | 821.0008206965983
average | 821.001
consumption | 13.9
rows | 2

and the value 821.001 is the one wich is display in volkszaehler frontend as last